packages/ui/src/components/avatar/btc-avatar-icon.native.tsx

```diff
@@ -0,0 +1,13 @@
+import BtcIcon from '../../assets/icons/bitcoin.svg';
+import { Icon } from '../../icons/icon/icon.native';
+import { Avatar } from './avatar.native';
+
+export function BtcAvatarIcon() {
```

Comment:
@fbwoolf Do you think this is enough to satisfy https://github.com/leather-io/issues/issues/290? 

If we accept a color to `<Avatar` then we can pass `Icon` and an `svg` as children like this

Reply:
Seems good, yep. 👍 Thx for expanding on it.